<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>2</width>
      <height>0</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <widget class="QLabel" name="credit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>781</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>By: Derbzzzzzz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="header">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>781</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>60</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Minesweeper</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNewGame">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>380</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>New Game</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="header_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>781</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select Difficulty</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBeginner">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Beginner</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIntermediate">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>300</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Intermediate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdvanced">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>420</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Advanced</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBack">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>520</y>
        <width>131</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="btnQuit">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>520</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
     <widget class="QLabel" name="endLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>9</y>
        <width>741</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>50</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>26</width>
        <height>26</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLay"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
